proposals/incubation/kubevirt.adoc

```diff
@@ -0,0 +1,147 @@
+
+## Community
+
+### Governance
+
+* [Governance policy](https://github.com/kubevirt/community/blob/master/GOVERNANCE.md)
+* [Membership policy](https://github.com/kubevirt/community/blob/master/membership_policy.md)
+
+As the project is in the process of growing beyond its initial group of contributors, we are working on adopting additional written governance rules.
+
+### SIGs
+
+* [SIG list](https://github.com/kubevirt/community/blob/master/sigs.yaml)
+
+Most of those SIGs are not yet independent committees. The Performance and Scale SIG has its own leadership and contributors, and both Documentation and Storage SIG should soon have the same. The community plan is to, as the community grows, foster dedicated contributors in each SIG area. As well as expanding our contribution pipeline, this will provide more opportunities for new contributors to rise to leadership of the project.
+
+### Maintainers
+
+* https://github.com/kubevirt/community/blob/master/MAINTAINERS.md
+
+### GitHub source code repository
+
+* About 50 contributors measured by multiple L or greater sized contributions
+* 171 official members in the organization
+* 2562 stars
+* 615 forks
+* 96 watched tags
+
+### Communications
+The project utilizes Google Groups as a [mailing list](https://groups.google.com/g/kubevirt-dev) where where users have a chance to interact with core developers and discuss general topics. There are currently (2021/06/01) 436 subscribers.  The [mailing list](https://groups.google.com/g/kubevirt-dev) typically receives 0-5 new threads per day.
+
+The project holds a video conference every week where users have a chance to interact with core developers, discuss general topics and participate in bug triage. Recently the performance and scale working group started conducting a weekly meeting based on the topic. Both meetings are recorded and posted to the project YouTube channel. Meeting notes are also emailed to the general [mailing list](https://groups.google.com/g/kubevirt-dev) to keep the community informed.
+
+The project uses two channels on the CNCF Kubernetes Slack server. #virtualization is used to handle general use topics. The channel hosts over 1000 users. #kubevirt-dev is used for developer oriented topics. The channel hosts 350 users. The channels typically receive 0-5 new threads per day.
+
+Important announcements are relayed via [mailing list](https://groups.google.com/g/kubevirt-dev), website blog and Twitter.
+
+KubeVirt advertises communications channels via https://kubevirt.io/community as well as project [README](https://github.com/kubevirt/kubevirt/blob/master/README.md).
+
+### Adopters
+
+The Kubevirt project tracks adopters via the [Adopters file](https://github.com/kubevirt/kubevirt/blob/main/ADOPTERS.md) files
```

Comment:
+1 on linking.

But please open a PR to structure the external document into end-users and non-end-users